### ISSUE TITLE
Error occurs on adding items to menu on Ubuntu 20.04

### ISSUE THREAD

**Issue by rkovatch** [lukearran/GNOME-Cast-Control#3]
I've just installed the extension on Ubuntu 20.04 and the menu displays "An error occurred..."

Running `journalctl /usr/bin/gnome-shell -f -o cat` shows the following being logged:

```
Google Cast Control 3: GET Request Started to Cast API at http://localhost:3000/device
Google Cast Control 3: Now refreshed with API... waiting until next 15000ms
Google Cast Control 3: An error occurred on adding items to the menu. Reverting to error view: Error: Invalid value 'undefined' for property state in object initializer.
```

I checked to see if `cast-web-api-cli` was working correctly, and it seems to be running and properly discovering my devices. Here is my environment:

Ubuntu 20.04 Focal Fossa
GNOME 3.36.0
Default extension settings
cast-web-api-cli 1.2.1
npm 6.3.17

Thanks!

**Comment by lukearran** [comment]
Thanks for reporting the issue,

Have you tried cloning this repsitory to `~/.local/share/gnome-shell/extensions/castcontrol@hello.lukearran.com` instead of using the version in the GNOME Extensions site? I've recently pushed a fix this, but haven't had the opportunity to thoroughly test it on previous versions.

Kindest Regards

Luke

**Comment by lukearran** [comment]
Just an update,

I'll be pushing a new version to GNOME Extensions site today, with the fix included for this issue on Ubuntu 20.04. It may take up to 7 days before it's approved on the site.

Kindest Regards

Luke
